### PR TITLE
feat: MAMORU-813:Collect Daemons statistics

### PR DESCRIPTION
**Pull request by gofmanaa**
added statistics

### CODE REVIEW

File: crates/mamoru-sui-sniffer/src/lib.rs [hunk 1]
```diff
@@ -80,7 +80,7 @@ impl SuiSniffer {
 
             let before_ms = Utc::now().timestamp_millis();
 
-            register_call_traces(ctx_builder.data_mut(), seq, call_traces);
+            register_call_traces(ctx_builder.data_mut(), seq, call_traces.clone());
```
Comment (sokyrko) [inline crates/mamoru-sui-sniffer/src/lib.rs]: Don't do clones, it takes too much memory.

File: crates/sui-transactional-test-runner/src/test_adapter.rs [hunk 2]
```diff
@@ -781,11 +781,12 @@ impl<'a> MoveTestAdapter<'a> for SuiTestAdapter<'a> {
                     let named_address_mapping = &mut self.compiled_state.named_address_mapping;
                     let dep = &Symbol::from(dep.as_str());
 
-                    let orig_package = if let Some(orig_package) = self.package_upgrade_mapping.get(dep) {
-                        orig_package
-                    } else {
-                        continue;
-                    };
+                    let orig_package =
```
Comment (sokyrko) [inline crates/sui-transactional-test-runner/src/test_adapter.rs:784]: Is this fmt?

Comment (gofmanaa) [reply crates/sui-transactional-test-runner/src/test_adapter.rs:784]: yes

File: crates/mamoru-sui-sniffer/src/lib.rs (hunk 1 above)
Comment (gofmanaa) [reply crates/mamoru-sui-sniffer/src/lib.rs]: fixed

**Review by sokyrko — APPROVED** [review]

**Pull request merged**